tests/args/test_common.py

```diff
@@ -114,3 +113,49 @@ def _(value):
 
     [which] = collect_dumped_objects(result)
     assert which == "re"
+
+
+def test_same_name_for_step_arg_and_example_parameter(pytester):
+    """Test that using the same name for step arg and example parameter as intended."""
```

Comment:
Missing the main clause in this sentence :)

Reply:
Whoopsy!